Libraries/CameraRoll/RCTAssetsLibraryRequestHandler.m

```diff
@@ -53,10 +53,20 @@ - (id)sendRequest:(NSURLRequest *)request
 
       ALAssetRepresentation *representation = [asset defaultRepresentation];
       NSInteger length = (NSInteger)representation.size;
+      
+      NSString *mimeType = nil;
+      switch (CGImageGetAlphaInfo([representation CGImageWithOptions:nil])) {
```

Comment:
Actually, I'm not sure this is a safe assumption. Just because an image doesn't have an alpha channel, that doesn't mean it's necessarily represented as a PNG.

Can you use MobileCoreServices to convert the UTI to a mimeType instead? There's a code example here: https://gist.github.com/ddeville/1527517